src/TipTapEditor/MenuBar.tsx

```diff
@@ -1,7 +1,12 @@
+import './MenuBar.css';
+
 import { Editor } from '@tiptap/react';
 import * as React from 'react';
 
+import { cx } from '../cx';
+
 export function MenuBar({ editor }: { editor: Editor }) {
+  const [counter, setCounter] = React.useState(1);
```

Comment:
If I understand correctly, `counter` is only used to force re-render, since you never use its value. Maybe you could create a `useForceUpdate` hook that does it, instead of having an unnecessary counter here. This way we could also use the hook in other components.
I also prefer a boolean state toggling between true and false to force re-render, instead of having a counter that could theoretically grow to infinite.

Reply:
You are right! I forgot this "hack". Not sure about the `useForceUpdate` hook. For now will update this to a boolean toggle.

I've had to implement this in order for the menu bar to update when the editor (state) changes.